convert/jenkinsjson/json/sh.go

```diff
@@ -6,9 +6,9 @@ import (
 	harness "github.com/drone/spec/dist/go"
 )
 
-func ConvertSh(node Node, variables map[string]string, timeout string, dockerImage string) *harness.Step {
+func ConvertSh(node Node, variables map[string]string, timeout string, dockerImage string, label string) *harness.Step {
 	shStep := &harness.Step{
-		Name:    SanitizeForName(node.SpanName),
+		Name:    SanitizeForName(node.SpanName) + label,
```

Comment:
You probably want to concatenate before passing into sanitise, otherwise a longer name will go over the Harness length limit.
It's not a major issue as the name can be easily manually trimmed but it does add another manual touch point.

Reply:
Ah yes thanks. I'll do that and see how what the output is like